### PR TITLE
Task jd/j270 output followers on the users page

### PR DESCRIPTION
**Pull request by JuliaDzyuba**
have some problem with query

### CODE REVIEW

File: frontend/src/components/user-followers/user-followers.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import styles from './styles.module.scss';
+import { useAppSelector, useParams, useDispatch, useEffect } from 'hooks/hooks';
+import { RootState } from 'common/types/types';
+import { PageParams } from './common/types/types';
+import { userFollowers as userFollowersActions } from 'store/actions';
+// import FollowerItem from './components/follower-item/follower-item';
```
Comment (Denidzio) [inline frontend/src/components/user-followers/user-followers.tsx]: Redundant commnet

File: frontend/src/components/user-followers/user-followers.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  useEffect(() => {
+    dispatch(userFollowersActions.loadFollowersByUserId(Number(id)));
+  }, [id]);
+
+  const { followers } = useAppSelector(({ userFollowers }: RootState) => ({
+    followers: userFollowers,
+  }));
+
+  // eslint-disable-next-line no-console
+  console.log('followers', followers);
```
Comment (Denidzio) [inline frontend/src/components/user-followers/user-followers.tsx]: Redundant `console.log`

File: frontend/src/components/user-followers/user-followers.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <div className={styles.container}>
+      <h2 className={styles.title}>Followers:</h2>
+      <div className={styles.followersList}>
+        {/* {followers.map((follower) => (
+          <FollowerItem key={follower.id} user={follower} />
+        ))} */}
```
Comment (Denidzio) [inline frontend/src/components/user-followers/user-followers.tsx]: You forgot to remove the comment

File: frontend/src/components/user-profile/user-profile.tsx [hunk 4]
```diff
@@ -133,27 +133,34 @@ const UserPage: React.FC = () => {
...
         )}
+        <div className={styles.followContainer}>
+          <Link
+            to={`${AppRoute.USERS}/${id}${AppRoute.FOLLOWERS}`}
+            className={styles.followCountContainer}
+          >
+            {/* <div className={styles.followCountContainer}> */}
+            <span className={styles.followCount}>{followersCount}</span>
+            <span className={styles.followTitle}>Followers</span>
+            {/* </div> */}
```
Comment (Denidzio) [inline frontend/src/components/user-profile/user-profile.tsx]: Also forgot to remove the comment

File: frontend/src/store/user-follower/actions.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import { AsyncThunkConfig, UserFollower } from 'common/types/types';
+import { ActionType } from './common';
+
+const loadFollowersByUserId = createAsyncThunk<UserFollower[], number, AsyncThunkConfig>(
+  ActionType.LOAD_FOLLOWERS_BY_USER_ID,
+  async (userId, { extra }) => {
+    const { userFollowerApi } = extra;
+    const followers = await userFollowerApi.getAllByUserId(userId);
+    // eslint-disable-next-line no-console
+    console.log('followers', followers);
```
Comment (Denidzio) [inline frontend/src/store/user-follower/actions.ts]: Redundant `console.log`

**Review by Denidzio — CHANGES_REQUESTED** [review]

**Review by EugeniusGR — DISMISSED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**